### ISSUE TITLE
Running multiple instances on one machine?

### ISSUE THREAD

**Issue by MRokas** [tananaev/traccar#1689]
There's pretty big chance I might be required to run two instances of Traccar on one VPN. Is that possible? At the moment I'm unable to test, so if it isn't, would sandboxing or re-compile(with changed code) work?

**Comment by tananaev** [comment]
It is possible. If you want both to run as system services, then you would have to modify java service wrapper configuration for the second service.

**Comment by MRokas** [comment]
I assume it's */conf/wrapper.conf* file, right? Thanks!

**Comment by tananaev** [comment]
Yes, and also you probably need to modify `/bin/traccar` script.

**Comment by MRokas** [comment]
Thanks for information!

I've to say, it's way easier than I expected it to be.

**Comment by zzyber** [comment]
did you get this up and going?
